modules/tensor_mechanics/src/interfacekernels/CZMInterfaceKernelSmallStrain.C

```diff
@@ -108,9 +106,9 @@ CZMInterfaceKernel::computeQpOffDiagJacobian(Moose::DGJacobianType type, unsigne
       break;
```

Comment:
The logic here is not quite right. Either I fix it in a separate PR or you fix it here.

You need to consider there might be other variables in the problem.

Reply:
I think I fixed it